components/script/dom/canvasrenderingcontext.rs

```diff
@@ -0,0 +1,1401 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's move this to `components/script/canvas_state.rs` instead.

Reply:
Right, I suggest we rename components/script/dom/canvasrenderingcontext.rs to components/script/canvas_state.rs.

Reply:
Hmmm, how is `components/script/dom/canvasrenderingcontext2d.rs` suppose to access `canvas_state` then? there is no `mod.rs` that exposes this

Reply:
components/script/lib.rs should include a `pub(crate) mod canvas_state;` line.